feat: hidden directories scan

guyeise5 opened the pull request:
Adding a command line flag to extend the file scan for hidden directories.

gagoar reviewed (commented):
Could you help me with some tests for this? Lemme know if you can follow what I've done before

gagoar commented:
Hey! I appreciate your help. Some things missing so we can merge your PR!

1) Tests.  Ensure you add some test that uses the feature you are adding. It will help us keep the code compatible as we move forward. 
2) Linting, please install the lining features so that prettier does its job! 
3) Rebase with main/master. I've removed some of the engines that we were testing against (12 and 14)

I will be on the lookout for your updates.

guyeise5 commented:
> Could you help me with some tests for this? Lemme know if you can follow what I've done before

Added some tests

> Linting, please install the lining features so that prettier does its job!

Tried, not sure I have done it correctly.

> Rebase with main/master. I've removed some of the engines that we were testing against (12 and 14)

done

@gagoar
